lib/openshift/cluster_logging.rb

```diff
@@ -0,0 +1,107 @@
+# represent a clusterlogging object
+module BushSlicer
+  class ClusterLogging < ClusterResource
+    RESOURCE = "clusterloggings"
+
+    ### represent status section ["collection", "curation", "logStore", "message", "visualization"]
+    def collection(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'collection')
+    end
+    def curation(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'curation')
+    end
+
+    def log_store(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'logStore')
+    end
+
+    def message(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'message')
+    end
+
+    def visualization(user: nil, cached: true, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'visualization')
+    end
```

Comment:
what values do these return?

Reply:
what I would like to is to have all low-level methods that return some raw YAML to be private. Users shouldn't parse raw structures. Then we can gradually add methods returning proper objects when needed.

Similar to how we have `QuotaLimits#memory_limit_raw` vs `QuotaLimits#memory_limit` methods. The raw method returns unprocessed structure. The non-raw method returns some more convenient object.

Reply:
Those are part of the `status` return values.  Unlike `QuotaLimits`, they are not values but rather `yaml/json` structs.  You can look at it here https://github.com/openshift-qe/output_references/blob/master/logging/4.x/clusterlogging.yaml

Reply:
I know, that's why I'm asking you to name methods `..._raw` and make `private` all that are never called outside the class. We can create deal with any external users when we have such.